test/framework/src/main/java/org/elasticsearch/index/mapper/TestDocumentParserContext.java

```diff
@@ -74,7 +74,7 @@ public final LuceneDocument doc() {
     }
 
     @Override
-    public final ContentPath path() {
+    public ContentPath path() {
```

Comment:
is there a specific reason for this change?

Reply:
I had to override this in `server/src/test/java/org/elasticsearch/index/mapper/DynamicFieldsBuilderTests.java`

Reply:
I think this can be avoided by removing the overwrite and add `ctx.path().add("labels");` at line 83 in `DynamicFieldsBuilderTests` just after creating `DocumentParserContext`.